src/app/group/group-add-edit/group-add-edit.component.ts

```diff
@@ -11,68 +14,164 @@ import { filter } from 'rxjs';
 export class GroupAddEditComponent implements OnInit {
 
   addGroupForm !: FormGroup;
+  addMembersForm !: FormGroup;
+  currentUser : string='';
   groupTypeArray: string[] = ['Trip', 'Couple', 'Other'];
   isGroupAddActive: Boolean = false;
-  getActivatedRouteParam : String = '';
+  getActivatedRouteParam: String = '';
+  isGroupCreated: boolean = false;
 
   // Constructor
-  constructor( private router: Router, private activatedRoute: ActivatedRoute){};
+  constructor(private router: Router,
+    private activatedRoute: ActivatedRoute,
+    private groupsService: GroupsService) {
+
+    /*******************************testing code*************************************/
+
+    // this.groupsService.logoutUser().subscribe((res)=> console.log(res))
+    // setTimeout(() => {
+    //   this.getLogin();
+    // }, 1000);
+
+    // setTimeout(() => {
+    //   this.getCurrentUser()
+    // }, 2000);
+
+  };
+
+
+
+
+
 
   // ngOnInit method
   ngOnInit(): void {
+    /** Form group for add group page
+     */
     this.addGroupForm = new FormGroup({
       groupName: new FormControl('', [Validators.required]),
-      shareLink: new FormControl(''),
-      groupType: new FormControl('Home'),
+      about: new FormControl(''),
       groupMember: new FormArray([
-        this.memberNameAndEmailForm()
+        new FormGroup({
+          userId: new FormControl('3a0ba79a-0f40-0dd1-4e91-9ed501777180')
+        })
       ]),
     })
 
+
+    // form group add members to group
+    this.addMembersForm = new FormGroup({
+    groupMembers: new FormArray([
+      this.addMembersToGroupForm()
+    ]),})
+
+
     // get activatedRoute parameter using observable
-    this.activatedRoute.params.subscribe((param) =>{
+    this.activatedRoute.params.subscribe((param) => {
       this.getActivatedRouteParam = param['routerParam'];
-      console.log(this.getActivatedRouteParam)
-      if(this.getActivatedRouteParam === undefined ){
+      // console.log(this.getActivatedRouteParam)
+      if (this.getActivatedRouteParam === undefined) {
         this.isGroupAddActive = true;
       }
-      else{
+      else {
         this.isGroupAddActive = false;
       }
     })
+
+
   }
 
   // Add member name and member email formgroup in add group form
-  memberNameAndEmailForm(): FormGroup{
+  addMembersToGroupForm(): FormGroup {
```

Comment:
follow method comment format

Reply:
Method comment format is not followed https://marketplace.visualstudio.com/items?itemName=salbert.comment-ts